src/main/java/seedu/address/ui/DeliveryCard.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.ui;
+
+import javafx.scene.layout.Region;
+import seedu.address.model.delivery.Delivery;
+
+/**
+ * An UI component that displays information of a {@code Item}.
+ */
+public class DeliveryCard extends UiPart<Region> {
+
+    private static final String FXML = "ItemListCard.fxml";
```

Comment:
Is this meant to be DeliveryListCard.fxml or sth along that line?

Reply:
thanks for catching that! yes it should be DeliveryListCard

Reply:
have fixed it